lib/hanami.rb

```diff
@@ -118,6 +118,30 @@ def self.app
     App.new(configuration, environment)
   end
 
+  # Check to see if application allowed to load.
+  #
+  # You can set allowed application in `HANAMI_APPS` env variable.
+  # If `HANAMI_APPS` variable is emply - any apps allowed.
```

Comment:
Suggestion: "If the `HANAMI_APPS` env variable is not set, it defaults to loading all applications."

Which brings the question of differentiating between explicitly setting `HANAMI_APPS=""` vs not setting it. It might be worth allowing to load no app at all.